README.md

```diff
@@ -472,6 +472,29 @@ Prints string to stdout, and returns string with additional utility methods
 like `.to()`.
 
 
+### mktemp([options,] [templates...])
+
+Available Options:
+
++ `-d`: Create a directory instead of a file.
++ `-u`: Unsafe mode. (Delete the file before retuning). Only use if you know what you're doing.
+
+Examples:
+
+```javascript
+mktemp() // ['/tmp/tmp.shelljs.QOL1QLNZMSR0HR4S5FTS']
+mktemp('-d') // ['/tmp/tmp.shelljs.EGBZ4GVYFO4SO534F3WK']
+mktemp('/tmp/tmp.shelljs.foo.XXXXX') // ['/tmp/tmp.shelljs.foo.AQZJK']
+mktemp('-d', '/tmp/shelljs.foo.XXXXX', '/tmp/shelljs.bar.XXXXX') // ['/tmp/tmp.shelljs.foo.ZFUPD', '/tmp/tmp.shelljs/bar.I9XVF']
```

Comment:
Does Mac `mktemp` allow more than one template? For me it only allows one on Linux.